pages/api/cptec/v1/clima/previsao/[cityCode]/[days].js

```diff
@@ -0,0 +1,37 @@
+import app from '@/app';
+import BadRequestError from '@/errors/BadRequestError';
+import NotFoundError from '@/errors/NotFoundError';
+import { getPredictionWeather } from '@/services/cptec';
+
+const action = async (request, response) => {
+  const { days, cityCode } = request.query;
+
+  if (days < 1 || days > 14) {
```

Comment:
O tipo do `days` não é uma string? Não precisa converter para int?

Reply:
Acho que vou fazer melhor, verificar se é número, e se não for tacar um badRequest no lombo do sujeito.. .rs

Reply:
Tratamento adicionado, só não sei se vale fazer o parse